src/components/DotsMenu/DotsMenu.js

```diff
@@ -33,7 +33,7 @@ export class DotsMenuComponent {
     const activeMenuItem = e.target.closest(".dots-menu__item");
     if (activeMenuItem) {
       const id = parseInt(activeMenuItem.dataset.id, 10);
-      this.props.items[id].handler();
+      this.props.items[id].handler(activeMenuItem);
```

Comment:
you should not pass an element here. You can pass a value. Other components should not have access to internals of this component